spec/new_inject_method_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'new_inject_method.rb'
+
+describe Array do
+
+  let (:test_array){[1,2,3,4]}
```

Comment:
Space missing after comma.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.